include/zephyr/bluetooth/hci.h

```diff
@@ -113,6 +113,16 @@ int bt_hci_cmd_send_sync(uint16_t opcode, struct net_buf *buf,
  */
 int bt_hci_get_conn_handle(const struct bt_conn *conn, uint16_t *conn_handle);
 
+/** @brief Get connection given a connection handle.
+ *
+ * @param handle The connection handle
+ * @param type   The connection handle type
```

Comment:
Where does the user get the `type` from? `bt_hci_get_conn_handle` does not return a type

Reply:
I would assume that the user knows the handle type based upon context. For example if there is an event that is specific to ACLs or specific to ISO, the user would know what type to use

Reply:
There is of course also just `BT_CONN_TYPE_ALL` which can be used, so this is probably a non-issue

Reply:
Yes. Some handle types are not overlapping. See Core_v5.4, Vol 4, Part E, Section 5.3.

We could simply use CONN_TYPE_ALL and not let the user specify the handle type. I would guess that this is simpler